schema/verify/roles/ehs-results-exporter/grants.sql

```diff
@@ -0,0 +1,12 @@
+-- Verify seattleflu/id3c-customizations:roles/ehs-results-exporter/grants on pg
+
+begin;
+
+select 1/pg_catalog.has_database_privilege('ehs-results-exporter', :'DBNAME', 'connect')::int;
+select 1/pg_catalog.has_schema_privilege('ehs-results-exporter', 'shipping', 'usage')::int;
+select 1/pg_catalog.has_table_privilege('ehs-results-exporter', 'shipping.uw_reopening_ehs_reporting_v1', 'select')::int;
+
+select 1/(not pg_catalog.has_table_privilege('ehs-results-exporter', 'shipping.uw_reopening_ehs_reporting_v1', 'insert,update,delete'))::int;
```

Comment:
this can be removed since the view specific grant needs to be in the views rework.

Reply:
I kept the non-view checks.